test_util/local.ts

```diff
@@ -53,6 +53,9 @@ export class LocalClientEffect extends Z.Effect<LocalClient, PolkadotBinNotFound
   }
 
   private async createClient(): Promise<LocalClient> {
+    // TODO: remove logging
+    console.log("creating client...");
```

Comment:
Added `nice console messaging from test ctx / spawned Polkadot processes` to our ongoing list in #215.